posthog/celery.py

```diff
@@ -40,6 +45,71 @@
 UPDATE_CACHED_DASHBOARD_ITEMS_INTERVAL_SECONDS = settings.UPDATE_CACHED_DASHBOARD_ITEMS_INTERVAL_SECONDS
 
 
+# following instructions from here https://django-structlog.readthedocs.io/en/latest/celery.html
+@setup_logging.connect
+def receiver_setup_logging(loglevel, logfile, format, colorize, **kwargs):  # pragma: no cover
+    logging.config.dictConfig(
```

Comment:
could we reduce the config we find here https://github.com/PostHog/posthog/blob/master/posthog/settings/logs.py#L18 ?